PerfTools/JeProf/interface/jeprof.h

```diff
@@ -0,0 +1,4 @@
+
+namespace jeprof {
```

Comment:
On further thought, maybe `cms::jeprof` name would be better to make it clear the function is specific to CMS.